tasks/sep/021-refresh-signers-fakeFuS/.env

```diff
@@ -0,0 +1,3 @@
+ETH_RPC_URL="https://ethereum-sepolia.publicnode.com"
```

Comment:
I'm not sure if having multiple directories called `021-*` in `/sep` tasks directory is what we want. Should it increment?

Reply:
Replace the task by `021` -> `023`.